docs/connectors/index.mdx

```diff
@@ -48,9 +46,20 @@ This architecture allows the user to combine different transports and data forma
 > **Note:** these basics apply to all connectors **except** the
```

Comment:
**Note:** Some transports, e.g., [Delta Lake](/docs/connectors/sources/delta) and [datagen](/docs/connectors/sources/datagen), use fixed predefined data formats and do not require the `format` section in the connector specification.